docs/hub/spaces-gpus.md

```diff
@@ -91,4 +91,26 @@ Spaces running on free hardware are suspended automatically if they are not used
 Additional lifecycle control over Spaces with upgraded hardware will soon be provided such as configuration of an automated suspension of the Space after a custom delay.
 </Tip>
 
-Additional information about billing can be found in the [dedicated section](./billing).
\ No newline at end of file
+Additional information about billing can be found in the [dedicated Hub-wide section](./billing).
+
+## Expose a custom sleep time[[sleep-time]]
+
+If your Space runs on the default `cpu-basic` hardware, it will go to sleep if inactive for more than a set time (currently, 72 hours). Anyone visiting your Space will restart it automatically.
+
+If you want your Space to never deactivate, or if you want to set a custom sleep time, you need to upgrade to a paid Hardware.
+
+By default, an upgraded Space will never go to sleep. However, you can use this setting for your upgraded Space to "turn off" (go back to cpu-basic) when it's unused. 😴
```

Comment:
Is it what is happening? Going back to `cpu-basic` when inactive? I thought it was in that inactive state that gets rebuilt once a visitor comes just like the 72h

Reply:
i'm actually not sure 🤷 

but yeah i think you're right, it transitions to "stopped" stage, i'll update